consul/files/config.hcl

```diff
@@ -1,4 +1,4 @@
-{%- set home = '/home/consul' %}
+{%- set home = '/etc/consul' %}
```

Comment:
`$HOME` itself should not change to `/etc/consul`, `/etc/consul` is only for config.